e2e/poms/formBuilder.ts

```diff
@@ -0,0 +1,124 @@
+        if (option === "Single") {
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.singleChoiceOptionsContainer).click();
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.addBulkOptionLink).click();
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.bulkAddOptionsTextarea).fill(choices);
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.bulkAddOptionsDoneButton).click();
+        } else {
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.multiChoiceOptionsContainer).click();
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.addBulkOptionLink).click();
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.bulkAddOptionsTextarea).fill(choices);
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.bulkAddOptionsDoneButton).click();
+        }
+    }
+
+    randomizeOptions = async (option: string) => {
+        if (option === "Single") {
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.singleChoiceOptionsContainer).click();
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.randomizeSwitchLabel).click();
+        } else {
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.multiChoiceOptionsContainer).click();
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.randomizeSwitchLabel).click();
+        }
+        await expect(this.page.getByTestId('properties-header').getByTestId('neeto-ui-spinner')).toBeVisible();
+        await expect(this.page.getByTestId('properties-header').getByTestId('neeto-ui-spinner')).toBeHidden();
+    }
+
+    hideQuestion = async (option: string) => {
+        if (option === "Single") {
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.singleChoiceOptionsContainer).click();
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.hideQuestionToggleLabel).click();
+        } else {
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.multiChoiceOptionsContainer).click();
+            await this.page.getByTestId(FORMBUILDER_SELECTORS.hideQuestionToggleLabel).click();
+        }
+        await expect(this.page.getByTestId('properties-header').getByTestId('neeto-ui-spinner')).toBeVisible();
+        await expect(this.page.getByTestId('properties-header').getByTestId('neeto-ui-spinner')).toBeHidden();
+    }
+
+    waitForPutThenGet = async (urlPattern = FORMBUILDER_API.publishAPI, timeout = 5000) => {
+        const waitForMethod = async (method: string) => 
+          this.page.waitForResponse(
+            res => res.request().method() === method && res.url().startsWith(urlPattern) && res.status() === 200,
+            { timeout }
+          );
+      
+        await waitForMethod('PUT');
+        return await waitForMethod('GET');
+    }
+    
+    multiOptionVisible = async () => {
+        return await this.page.getByTestId(FORMBUILDER_SELECTORS.multiChoiceOptionsContainer).isVisible();
+    }
```

Comment:
Avoid redundant awaits:

```suggestion
    multiOptionVisible = this.page.getByTestId(FORMBUILDER_SELECTORS.multiChoiceOptionsContainer).isVisible;
```